### ISSUE TITLE
Scrap a site iso-8859-1 Ar�bia

### ISSUE THREAD

**Issue by phstc** [SaltwaterC/http-request#5]
Hello

I'm trying to scrap a site with iso-8859-1

```
    <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
```

I'm getting � for the accents. How can I configure the encoding for iso-8859-1?

Cheers,
Pablo Cantero

**Comment by SaltwaterC** [comment]
Hello,

For the moment, the encoding is not supported. The toString() method of the chunk buffer defaults the encoding to utf-8. You can save the raw response to a file, but I guess this is not what you want. http-get requires a patch to be able to pass a specific encoding if you're buffering the response body to a String object. I have plans to implement something in order to be able to return the buffer as Buffer object which would also fit the task.

Regards,
Stefan

**Comment by phstc** [comment]
Hi @SaltwaterC 

Tks for the reply.

Do you know other module that I can do it? I had this same problem with request module. :/

Regards,
Pablo Cantero

**Comment by SaltwaterC** [comment]
Closed by relasing v0.4:
- New option: bufferType for specifying the fact that the library should return the buffers as Buffer instance instead of String instance.
- New option: encoding for specifying the encoding of the returned buffers, when returning the buffered data as string. Defaults to 'utf8'.

With this release people get generic buffer instances or strings in the designated encoding.
